staging/src/k8s.io/kubectl/pkg/cmd/cp/cp_test.go

```diff
@@ -290,7 +332,7 @@ func TestTarUntar(t *testing.T) {
 			fileType: RegularFile,
 		},
 		{
-			name:     "some/other/directory/",
+			name:     "l295some/other/directory",
```

Comment:
was the l295 prefix intentional? should we add a test case without a trailing slash rather than modify this testcase?

Reply:
yikes, not intentional, fixing...